.github/workflows/ci-java.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - name: 'Setup JDK'
         uses: actions/setup-java@v1
         with:
-          java-version: '1.8.0.252'
+          java-version: '1.11'
```

Comment:
looking around online and I think for java version 10 and higher, the '1.X' has bee dropped in favor of just 'X', for example, instead of '1.11' just using '11'. I see both in this PR but wondering if it all still works if we move entirely to just using '11' for the java version.